internal/pathpolicy/path_policy.go

```diff
@@ -24,6 +24,13 @@ func NewPathPolicies(entries map[string]PathPolicy) *PathPolicies {
 	return NewPathTrieFromMap(noType)
 }
 
+func mergePolicyPath(entry_1 map[string]PathPolicy, entry_2 map[string]PathPolicy) map[string]PathPolicy {
+	for k, v := range entry_2 {
+		entry_1[k] = v
+	}
+	return entry_1
+}
+
```

Comment:
Combined with the usage below, this will mutate the global default policy (`entry_1`) and basically add a `deny` rule for `/ostree` to the default.

Reply:
Fixed, add if does not exists